apps/token-manager/app/src/screens/Details.js

```diff
@@ -0,0 +1,193 @@
+import React, { useCallback } from 'react'
+import {
+  Accordion,
+  Bar,
+  BackButton,
+  Box,
+  Button,
+  GU,
+  Split,
+  formatTokenAmount,
+  textStyle,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+import SummaryBar from '../components/SummaryBar'
+import SummaryRows from '../components/SummaryRows'
+import VestingInfoBoxes from '../components/VestingInfoBoxes'
+import { useAppLogic, toISODate, useVestedTokensInfo } from '../app-logic'
+import { format, formatDistanceStrict, parseISO } from 'date-fns'
+
+const formatDate = date => `${format(date, 'do MMM yyyy, HH:mm O')}`
```

Comment:
We should move this to a `date-utils` or `utils` file :).

Also, I think we should discuss the date format. Elsewhere, we've settled on a different one (https://github.com/aragon/aragon-apps/pull/1103#discussion_r406679045).